obal/data/modules/check_copr_build.py

```diff
@@ -0,0 +1,54 @@
+"""
+Check if a build already exists in Copr
+"""
+
+import json
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.copr import copr_cli, CoprCliCommandError, full_name # pylint:disable=import-error,no-name-in-module
+
+
+def main():
+    """
+    Check if a build already exists in Copr
+    """
+    module = AnsibleModule(
+        argument_spec=dict(
+            user=dict(type='str', required=True),
+            project=dict(type='str', required=True),
+            nvr=dict(type='str', required=True),
+            package=dict(type='str', required=True),
+            config_file=dict(type='str', required=False),
+        )
+    )
+
+    user = module.params['user']
+    project = module.params['project']
+    nvr = module.params['nvr']
+    package = module.params['package']
+    config_file = module.params['config_file']
+
+    command = [
+        'get-package',
+        full_name(user, project),
+        '--name',
+        package,
+        '--with-all-builds'
+    ]
+
+    try:
+        package_info = json.loads(copr_cli(command, config_file=config_file))
+    except CoprCliCommandError as error:
+        if "Error: No package with name {} in copr {}".format(package, project):
```

Comment:
this is `if <a string>:` which is always `true`, did you forget to match against some output?

Reply:
Rather than matching here, can we catch this in `copr_cli` and implement a specific `CoprPackageRetrievalError` exception (which subclasses `CoprCliCommandError`?